lang/c.md

```diff
@@ -14,11 +14,23 @@ int main()
 }
 ```
 
-## What is C?
+## ❓ What is C?
 
 C programming is a general-purpose, procedural, imperative computer programming language developed in 1972 by Dennis M. Ritchie at the Bell Telephone Laboratories to develop the UNIX operating system. C programming is considered as the base for other programming languages, that is why it is known as mother language. C language is a system programming language because it can be used to do low-level programming (for example driver and kernel). It is generally used to create hardware devices, OS, drivers, kernels, etc. For example, Linux kernel is written in C. It supports the feature of both low-level and high-level languages.
+<br><br>
 
-## Why one should use C?
+## 🖱️ Application of C
+
+-   Operating Systems (UNIX, Linux, Microsoft Windows)
+-   Embedded Systems (Firmwares, Drivers)
+-   GUI (Adobe Photoshop, Adobe Premiere, Illustrator)
+-   New Programming Platforms (C++, MATLAB, Mathematica)
+-   Compiler Design (Dev-C, Clang C, MINGW, and Apple C)
+-   DataBase Server (MySQL)
+-   Gaming and Animation
```

Comment:
```suggestion
- Operating Systems (UNIX, Linux, Microsoft Windows)
- Embedded Systems (Firmwares, Drivers)
- GUI (Adobe Photoshop, Adobe Premiere, Illustrator)
- New Programming Platforms (C++, MATLAB, Mathematica)
- Compiler Design (Dev-C, Clang C, MINGW, and Apple C)
- Database Server (MySQL)
- Gaming and Animation
```

Reply:
@Panquesito7   Actually other indentation of bullet marks in this file are similar like that. I think It was done to clarify them in the raw script. To match them I also have done the same.

Reply:
> @Panquesito7 Actually other indentation of bullet marks in this file are similar like that. I think It was done to clarify them in the raw script. To match them I also have done the same.

Yep, I noticed that, but according to the Markdown linter, that's not correct, so please change it. 🙂

Reply:
okay